spec/spec_helper.rb

```diff
@@ -90,6 +90,11 @@
     SpreeAvataxOfficial::Config.enabled = false
   end
 
+  config.before do
+    Rails.cache.clear
+    SpreeAvataxOfficial::Config.company_code = 'test1'
```

Comment:
@Vegann this is already set here: https://github.com/spree-contrib/spree_avatax_official/blob/master/spec/support/avatax.rb#L6

Reply:
@Vegann also when clearing out cache we should clear out spree preferences, eg .

https://github.com/spree/spree_dev_tools/blob/8a451b6d0bfcc2d917379447dee818c267a6e301/lib/spree_dev_tools/rspec/support/spree.rb#L42

Reply:
@damianlegawiec Weird because, without setting `SpreeAvataxOfficial::Config.company_code = 'test1'` in `config.before` lots of tests are failing because they are trying to request `https://sandbox-rest.avatax.com/api/v2/companies//` instead of `https://sandbox-rest.avatax.com/api/v2/companies/test1/`

> also when clearing out cache we should clear out spree preferences, eg .

Good to know, I will add that